dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/AbstractJdbcTask.java

```diff
@@ -97,8 +100,8 @@ protected CSVFormat newCsvFormat(@Nonnull ResultSet rs) throws SQLException {
   }
 
   @Nonnull
-  protected ResultSetExtractor<Void> newCsvResultSetExtractor(
-      @Nonnull ByteSink sink, @CheckForSigned long count) {
+  protected ResultSetExtractor<Summary> newCsvResultSetExtractor(
+      @Nonnull ByteSink sink, @CheckForSigned long count, @Nullable ZonedInterval interval) {
```

Comment:
I would suggest to use `Optional<ZonedInterval>` type instead of `@Nullable ZonedInterval` for the `interval` parameter. If this is not an external API (JDBC, RPC/REST calls), null value should be avoided.